queries.sql

```diff
@@ -1,7 +1,22 @@
 -- Part 1: Test it with SQL
 
+-- Columns [Data type]
+-- id [INT]
+-- employer [VARCHAR(255)]
+-- name [VARCHAR(255)]
+-- skills [VARCHAR(255)]
+
+
 -- Part 2: Test it with SQL
 
+-- SELECT *
+-- FROM employer
+-- WHERE (location = "St. Louis City");
```

Comment:
```suggestion
-- SELECT *
-- FROM employer
-- WHERE location = "St. Louis City";
```

Your query is pretty close to being correct. Instead of listing all of the data from the employer table, you'll want to only list the name of the employer where the location is St. Louis City. What would you use instead of  `*` to target that specific column?

Also, you don't need to use parenthesis around `location = "St. Louis City".